README.md

```diff
@@ -90,6 +90,34 @@ Properties:
 - `transforms` - Map of name to `.transform()` functions
 - `preprocesses` - Map of name to `z.preprocess()` functions
 
+## Use of JSON References
+
+JSON references are used to represent local references. If you wish to use
+JSON references for remote references, you may do so, but you will need
+to use a library like [`json-refs`](https://github.com/whitlockjc/json-refs)
+(with `resolveRefs`) to first resolve such references and then supply the object
+to `dezerialize`.
+
```

Comment:
is that still the case?

Reply:
Yes, it's still the case. Btw, I added another caveat on the README.

Reply:
To clarify further...one must use an external JSON refs library to resolve things like:

```json
{"$ref': "https://example.com/some/Zodex/JSON/file.json"}
```

or

```json
{"$ref': "/a/Zodex/JSON/file/at/root.json"}
```

but not for internal references like:

```json
{"$ref': "#/somewhere/in/the/document"}
```